e2e/e2e-test-setup.js

```diff
@@ -0,0 +1,26 @@
+import { execSync } from 'child_process'
+import * as path from 'path'
+import { fileURLToPath } from 'url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
```

Comment:
We are on node 22, maybe we can consider access `import.meta` properties to read this:
- [import.meta.filename](https://nodejs.org/api/esm.html#importmetadirname) 
- [import.meta.dirname](https://nodejs.org/api/esm.html#importmetafilename)

Reply:
Fixed, thanks 👌